repair/repair.cc

```diff
@@ -570,7 +570,9 @@ repair::shard_repair_task_impl::shard_repair_task_impl(tasks::task_manager::modu
     , cfs(get_table_names(db.local(), table_ids_))
     , table_ids(std::move(table_ids_))
     , global_repair_id(parent_id_)
-    , data_centers(data_centers_)
+    , data_centers(boost::copy_range<std::vector<locator::dc_name>>(data_centers_ | boost::adaptors::transformed([] (auto& dc) {
+            return locator::dc_name(std::move(dc));
+        })))
     , hosts(hosts_)
```

Comment:
out of curiosity, why do we need this copy_range? data_centers_ is const ref. doesn't the old copy work? locator::dc_name is copyable, no?

Reply:
good catch, probably left over from when `data_centers_` was a `std::vector<locator::dc_name>`.
It can be dropped.